src/components/inline/Card/Card.module.css

```diff
@@ -0,0 +1,39 @@
+.card-container {
+  height: auto;
+  width: 370px;
+}
+
+.img-container {
+  margin-bottom: 20px;
+}
+
+.img {
+  height: 250px;
+  width: 100%;
+}
+
+.title-container {
+  margin-bottom: 10px;
+}
+
+.title {
+  color: var(--color-white);
+  font-family: var(--font-family-title);
+  font-size: 30px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 130%;
+}
+
+.description-container {
+  margin-bottom: 20px;
```

Comment:
I think we should take out this bottom margin. In case if you're not familiar with collapsing margins, [here's an article about them](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing). Without going into the details, what this means for us is that this margin effectively gets applied to the whole card component, not just this `description-container`, which means that this margin exists _below_ the entire card component.

Here's a screenshot of what this looks like with the Chrome inspector, where the orange region is the margin:

<img width="431" alt="Screen Shot 2020-10-05 at 10 09 40 PM" src="https://user-images.githubusercontent.com/1002748/95161264-8895cd00-0757-11eb-813b-f059fe7cd4a5.png">


I would advise not doing this because this could make it more difficult to position the card in scenarios where we do not want that bottom margin.